pipeline/metadata/flatten_satellite.py

```diff
@@ -388,53 +388,59 @@ def _process_satellite_v2p2(self, row: Row,
     """
     responses = responses_entry.get('response', [])
     row['controls_failed'] = not responses_entry['passed_liveness']
```

Comment:
Maybe call this `base_row` to be extra explicit?

Reply:
done